1-js/03-code-quality/04-ninja-code/article.md

```diff
@@ -1,147 +1,147 @@
-# Ninja code
+# निंजा कोड
 
 
-```quote author="Confucius (Analects)"
-Learning without thought is labor lost; thought without learning is perilous.
+```उद्धरण लेखक = "कन्फ्यूशियस (अनालेक्ट्स)"
```

Comment:
These are smart tags, used by the page generator to generate the doc site. The site generator will fail if we change them.